main.go

```diff
@@ -1,25 +1,72 @@
 package main
 
 import (
-    "notes-service/grpc/notespb"
-    "net"
-    
-    "fmt"
+	"context"
+	"fmt"
+	"net"
+	"time"
+
+	"notes-service/grpc/notespb"
+
+	"google.golang.org/grpc"
+
+	"go.mongodb.org/mongo-driver/mongo"
+	option "go.mongodb.org/mongo-driver/mongo/options"
+	"go.mongodb.org/mongo-driver/mongo/readpref"
+
+	log "github.com/sirupsen/logrus"
 )
 
+var NotesDatabase *mongo.Database = nil
+var NotesCollection *mongo.Collection = nil
+
+// Need to put these variables in .env
+var password = "Gyy628\\nAWS"
+var mongoUri string = "mongodb+srv://gabriel:" + password + "@cluster0.2ckb3.mongodb.net/myFirstDatabase?retryWrites=true&w=majority"
```

Comment:
😂
Just saying, the repo is open-source, so you should **urgently** change the creds to your mongo cluster.

Reply:
Also check the `accounts-service` there is a `make run-db` rule which starts a local mongo database using Docker. In my opinion it's much nicer than connecting to a remote cluster for local development. You can then access the contents of the DB using `docker exec -it <container_id> /bin/sh` or by using a tool like DataGrip.